operator.go

```diff
@@ -927,7 +927,6 @@ func (o *operator) stepName(i int) string {
 
 func (o *operator) expand(in interface{}) (interface{}, error) {
 	store := o.store.toMap()
-	store["string"] = func(in interface{}) string { return fmt.Sprintf("%v", in) }
```

Comment:
I didn't know there was such a thing as a string function. I just noticed. 😅 
I think it's a good feeling to have it officially implemented. 👍